geolocation/android/src/main/java/com/capacitorjs/plugins/geolocation/GeolocationPlugin.java

```diff
@@ -51,6 +51,26 @@ protected void handleOnResume() {
         }
     }
 
+    @Override
+    @PluginMethod
+    public void checkPermissions(PluginCall call) {
+        if (implementation.isLocationServicesEnabled()) {
+            super.checkPermissions(call);
+        } else {
+            call.reject("System location services are not enabled");
+        }
+    }
+
+    @Override
+    @PluginMethod
+    public void requestPermissions(PluginCall call) {
+        if (implementation.isLocationServicesEnabled()) {
+            super.requestPermissions(call);
+        } else {
+            call.reject("System location services are not enabled");
```

Comment:
```suggestion
            call.reject("Location services are not enabled");
```
Match the current iOS message